index.js

```diff
@@ -6,21 +6,24 @@
 
 var Path = require('path');
 var Hapi = require('hapi');
+var vision = require('vision');
+var inert = require('inert');
 
 var handlebars = require('handlebars');
 handlebars = require('handlebars-helper-sri').register(handlebars);
 
 var helpers = require('./lib/helpers.js');
 
-var CSP_HEADER = "default-src 'none'; img-src 'self'; style-src 'self'; font-src 'self' ; frame-src 'self'"; // jshint ignore:line
-
 var server = new Hapi.Server();
+
+var CSP_HEADER = 'default-src "none"; img-src "self"; style-src "self"; font-src "self"; child-src "self"';
```

Comment:
Safari and Edge don't support CSP 2 but support CSP 1 so we should leave `frame-src` in there for backwards compatibility.

Also `frame-src` has been [un-deprecated in CSP3](https://w3c.github.io/webappsec-csp/#directive-frame-src).